code/game/objects/effects/spawners/costume_bundles.dm

```diff
@@ -78,131 +96,151 @@
-		/obj/item/clothing/suit/apron)
+		/obj/item/clothing/suit/apron,
+	)
 
 /obj/effect/spawner/bundle/costume/pirate
 	name = "pirate costume spawner"
 	items = list(
 		/obj/item/clothing/under/costume/pirate,
 		/obj/item/clothing/suit/pirate,
 		/obj/effect/spawner/lootdrop/minor/pirate_or_bandana,
-		/obj/item/clothing/glasses/eyepatch)
+		/obj/item/clothing/glasses/eyepatch,
+	)
 
 /obj/effect/spawner/bundle/costume/commie
 	name = "commie costume spawner"
 	items = list(
 		/obj/item/clothing/under/costume/soviet,
-		/obj/item/clothing/head/ushanka)
+		/obj/item/clothing/head/ushanka,
+	)
 
 /obj/effect/spawner/bundle/costume/imperium_monk
 	name = "imperium monk costume spawner"
 	items = list(
 		/obj/item/clothing/suit/imperium_monk,
-		/obj/effect/spawner/lootdrop/minor/twentyfive_percent_cyborg_mask)
+		/obj/effect/spawner/lootdrop/minor/twentyfive_percent_cyborg_mask,
+	)
 
 /obj/effect/spawner/bundle/costume/holiday_priest
 	name = "holiday priest costume spawner"
-	items = list(
-		/obj/item/clothing/suit/chaplainsuit/holidaypriest)
+	items = list(/obj/item/clothing/suit/chaplainsuit/holidaypriest)
 
 /obj/effect/spawner/bundle/costume/marisawizard
 	name = "marisa wizard costume spawner"
 	items = list(
 		/obj/item/clothing/shoes/sneakers/marisa,
 		/obj/item/clothing/head/wizard/marisa/fake,
-		/obj/item/clothing/suit/wizrobe/marisa/fake)
+		/obj/item/clothing/suit/wizrobe/marisa/fake,
+	)
 
 /obj/effect/spawner/bundle/costume/cutewitch
 	name = "cute witch costume spawner"
 	items = list(
 		/obj/item/clothing/under/dress/sundress,
 		/obj/item/clothing/head/witchwig,
-		/obj/item/staff/broom)
+		/obj/item/staff/broom,
+	)
 
 /obj/effect/spawner/bundle/costume/wizard
 	name = "wizard costume spawner"
 	items = list(
 		/obj/item/clothing/shoes/sandal,
 		/obj/item/clothing/suit/wizrobe/fake,
 		/obj/item/clothing/head/wizard/fake,
-		/obj/item/staff)
+		/obj/item/staff,
+	)
 
 /obj/effect/spawner/bundle/costume/sexyclown
 	name = "sexy clown costume spawner"
 	items = list(
 		/obj/item/clothing/mask/gas/sexyclown,
-		/obj/item/clothing/under/rank/civilian/clown/sexy)
+		/obj/item/clothing/under/rank/civilian/clown/sexy,
+	)
 
 /obj/effect/spawner/bundle/costume/sexymime
 	name = "sexy mime costume spawner"
 	items = list(
 		/obj/item/clothing/mask/gas/sexymime,
-		/obj/item/clothing/under/rank/civilian/mime/sexy)
+		/obj/item/clothing/under/rank/civilian/mime/sexy,
+	)
 
 /obj/effect/spawner/bundle/costume/mafia
 	name = "black mafia outfit spawner"
 	items = list(
 		/obj/item/clothing/head/fedora,
 		/obj/item/clothing/under/suit/blacktwopiece,
-		/obj/item/clothing/shoes/laceup)
+		/obj/item/clothing/shoes/laceup,
+	)
 
 /obj/effect/spawner/bundle/costume/mafia/white
 	name = "white mafia outfit spawner"
 	items = list(
 		/obj/item/clothing/head/fedora/white,
 		/obj/item/clothing/under/suit/white,
-		/obj/item/clothing/shoes/laceup)
+		/obj/item/clothing/shoes/laceup,
+	)
```

Comment:
indent once for all of these

Reply:
Fixed with c69817e4c6b8d6c2a0b77a6bccd8541ced713434